Genius/ContentPoster.cs

```diff
@@ -3,25 +3,20 @@
 using System.Net.Http.Headers;
 using System.Text;
 using System.Threading.Tasks;
+using Genius.Data;
 using Newtonsoft.Json;
 
 namespace Genius
 {
+	/// <summary>
+	/// Used to post content on behalf of the currently signed in user. 
+	/// </summary>
     public class ContentPoster
     {
-        /// <summary>
-        /// Many API requests accept a text_format query parameter that can be used to specify how text content is formatted.
-        /// The value for the parameter must be one or more of plain, html, and dom. 
-        /// The value returned will be an object with key-value pairs of formats and results:
-        ///
-        ///`plain` is just plain text, no markup
-        ///`html` is a string of unescaped HTML suitable for rendering by a browser
-        ///`dom` is a nested object representing and HTML DOM hierarchy that can be used to programmatically present structured content
-        /// </summary>
-        public static string TextFormat { get; set; }
-
-        public static string AuthorizationToken { get; set; }
-
+	    /// <summary>
+	    /// The token that is used for authorization.
```

Comment:
No need to state the obvious in comments. AuthorizationToken self explains what it is